src/Rings/MPolyQuo.jl

```diff
@@ -11,7 +11,8 @@
   SQRGB::Singular.sideal # Singular Groebner basis defining quotient ring
 
   function MPolyQuoRing(R::MPolyRing, I::MPolyIdeal, ordering::MonomialOrdering = default_ordering(R))
-    @assert base_ring(I) === R
+    @req base_ring(I) === R "Base ring must be the same."
```

Comment:
Unfortunately, this change causes problems in some old tests. Can you please stick to the old code line here.

Reply:
I mean this change:

@assert base_ring(I) === R
@req base_ring(I) === R "Base ring must be the same."